src/Cms/Language.php

```diff
@@ -474,9 +483,9 @@ public function toArray(): array
 	}
 
 	/**
-	 * Returns the translation strings for this language
+	 * Returns the language translations object for this language
 	 */
-	public function translations(): array
```

Comment:
Maybe we could keep returning an array here for backwards-compatibility and add an internal `translationsObject` method. This would allow shipping this feature in 4.x already as it seems to be a kind-of regression in 4.0. We could then change it to return the object in v5 with the breaking change and remove the internal method again.

Reply:
I thought about the new `->translationsObject()` method but I thought it would cause more confusion. I will try to implement it in a separate PR.

Reply:
Here K4 version: https://github.com/getkirby/kirby/pull/6216